src/actions/palette.js

```diff
@@ -6,3 +8,31 @@ export function setPaletteColor(name, colorCode) {
     payload: { name, colorCode },
   };
 }
+
+export function changePaletteColor(name, colorCode) {
+  return (dispatch) => {
+    const linkContainColorName = doesContinParam(name);
+    const separator = window.location.href.includes('?') ? '&' : '?';
+    const newLink = linkContainColorName
+      ? replaceUrlDataWith(name, colorCode)
+      : `${window.location.href + separator + name}=${colorCode}`;
```

Comment:
Let's use [URL](https://developer.mozilla.org/en/docs/Web/API/URL) to construct urls

Reply:
`#` must be either removed from url or encoded before setting new location.
Personally, i feel like having `#` symbol is redundant because we do not support any other color format.